tests/test_docker_compose.py

```diff
@@ -10,33 +10,33 @@ def p(*paths):
 class TestDockerCompose:
```

Comment:
When merging this PR, let's try:
1. `pytest` the **old** version of this file on the **old** codebase and make sure it **passes**.
2. `pytest` the **new** version of this file on the **old** codebase and make sure it **fails**.
3. `pytest` the **old** version of this file on the **new** codebase and make sure it **fails**.
4. `pytest` the **new** version of this file on the **new** codebase and make sure it **passes**.

**Optional**: run `ruff --select=PTH tests/test_docker_compose.py` and make sure it passes.